packages/web-twig/src/Resources/components/Select/Select.twig

```diff
@@ -0,0 +1,88 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _id = props.id | default(null) -%}
+{%- set _name = props.name | default(null) -%}
+{%- set _value = props.value | default(null) -%}
+{%- set _label = props.label | default(null) -%}
+{%- set _isFluid = props.isFluid | default(false) | boolprop -%}
+{%- set _isDisabled = props.isDisabled | default(false) | boolprop -%}
+{%- set _isLabelHidden = props.isLabelHidden | default(false) | boolprop -%}
+{%- set _isRequired = props.isRequired | default(false) | boolprop -%}
+{%- set _validationState = props.validationState | default(null) -%}
+{%- set _validationText = props.validationText | default(null) -%}
+{%- set _unsafeValidationText = props.UNSAFE_validationText | default(null) -%}
+{%- set _helperText = props.helperText | default(null) -%}
+{%- set _unsafeHelperText = props.UNSAFE_helperText | default(null) -%}
+{%- set _iconName = props.iconName | default('chevron-down') -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Select' -%}
```

Comment:
Please do not use this. We try to have whole classnames everywhere. Only exception is TextFieldBase which is an abstract component for TextField and TextArea.

See other components and modify this section to 
```
{%- set _rootFluidClassName = _isFluid ? 'Select--fluid' : null -%}
{%- set _rootDisabledClassName = _isDisabled ? 'Select--disabled' : null -%}
...
```

Reply:
see https://github.com/lmc-eu/spirit-design-system/pull/801#discussion_r1178309967